bin/helper/osh-groupDelete

```diff
@@ -110,15 +125,18 @@ my @tarcmd = qw{ tar czf };
 push @tarcmd, $fulldir . '.tar.gz';
 push @tarcmd, '--acls' if OVH::Bastion::has_acls();
 push @tarcmd, '--one-file-system', '-p', '--remove-files', $fulldir;
+
 $fnret = OVH::Bastion::execute(cmd => \@tarcmd, must_succeed => 1);
 if (!$fnret) {
-    osh_warn("Couldn't tar the backup homedir of this group (" . $fnret->msg . "), proceeding anyway.");
+    warn_syslog("Couldn't tar the backup homedir of this group (" . $fnret->msg . "), proceeding anyway.");
+    $nbErrors++;
     chmod 0000, $fulldir;
 }
 else {
     chmod 0000, $fulldir . '.tar.gz';
     unlink($fulldir);
```

Comment:
You can't unlink a directory

Reply:
This unlink was in fact not needed: the folder is no longer supposed to exist here, replaced by an existence test